### PR TITLE
[OSSM-3538] removed duplicate scenario names

### PR DESCRIPTION
**Pull request by ScriptingShrimp**
Polarion cannot process generated XML files from Cypress due to duplicate names of scenarios. This change removes all duplicate scenario names. Functionality was not changed.

### CODE REVIEW

File: frontend/cypress/integration/featureFiles/app_details.feature [hunk 1]
```diff
@@ -37,7 +37,7 @@ Feature: Kiali App Details page
     Then user sees trace details
 
   @app-details-page
-  Scenario: See span info after selecting a span
+  Scenario: See trace info after selecting a span
```
Comment (jshaughn) [inline frontend/cypress/integration/featureFiles/app_details.feature:40]:
```suggestion
  Scenario: See span info after selecting app span
```

Comment (jshaughn) [reply frontend/cypress/integration/featureFiles/app_details.feature:40]: And possibly a similar change in the conflicting service_details.feature.

File: frontend/cypress/integration/featureFiles/kiali_homepage.feature [hunk 2]
```diff
@@ -3,6 +3,6 @@ Feature: Kiali login
   I want to login to Kiali and see landing page
 
   @smoke  
-  Scenario: Open Kaili home page
+  Scenario: Kaili is present in the page title
```
Comment (jshaughn) [inline frontend/cypress/integration/featureFiles/kiali_homepage.feature:6]:
```suggestion
  Scenario: Kiali is present in the page title
```

**Review by jshaughn — CHANGES_REQUESTED** [review]
A lot of good catches here!  Just a couple of suggested changes so that the Scenario still accurately describes the test.

File: frontend/cypress/integration/featureFiles/kiali_homepage.feature (hunk 2 above)
Comment (ScriptingShrimp) [reply frontend/cypress/integration/featureFiles/kiali_homepage.feature]: fixed

File: frontend/cypress/integration/featureFiles/istio_config_editor.feature [hunk 3]
```diff
@@ -8,7 +8,7 @@ Feature: Kiali Istio Config editor page
     And user is at the "istio" page
     And user selects the "bookinfo" namespace
 
-  Scenario: Filter Istio Config objects by Valid configuration
+  Scenario: Verify editor loads correctly
```
[the remark this answers is not in the document]
Comment (ScriptingShrimp) [reply frontend/cypress/integration/featureFiles/istio_config_editor.feature]: fixed

File: frontend/cypress/integration/featureFiles/app_details.feature (hunk 1 above)
Comment (ScriptingShrimp) [reply frontend/cypress/integration/featureFiles/app_details.feature]: fixed both files

**Comment by jshaughn** [comment]
@ScriptingShrimp , OSSM v2.3.x is the Kiali 1.57 branch.

**Comment by ScriptingShrimp** [comment]
I removed last commit with unnecessary edits. XML output have been tested and it works now with Polarion.

**Review by jshaughn — APPROVED** [review]
Looks good!

**Pull request merged**